Makefile.toml

```diff
@@ -1423,9 +1434,25 @@ script = [
    '''
 ]
 
-# This task will call watch on the `status` testsys command to show the results of a test.
+# This task will call watch on the `status` testsys command to show the results of all tests.
+# To see all passed tests use `cargo make watch-test --passed`
+# To see all failed tests use `cargo make watch-test --failed`
+# To see all incomplete tests use `cargo make watch-test --running`
 [tasks.watch-test]
 dependencies = ["test-tools"]
+script = [
+   '''
+   set -eu
+   export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
+   watch -- testsys ${CARGO_MAKE_TESTSYS_ARGS} status --test ${@}
+   '''
+]
```

Comment:
Should the watch be optional? Rename the task and observe a `--watch` flag?

Reply:
I don't think so. Having that watch built in is super helpful. Users can do `cargo make testsys status` if they don't want it in `watch`.